gst/nnstreamer/tensor_converter/tensor_converter.c

```diff
@@ -1250,6 +1283,48 @@ gst_tensor_converter_parse_octet (GstTensorConverter * self,
   return (config->info.type != _NNS_END);
 }
 
+/**
+ * @brief Get sink pad template caps
+ */
+static GstCaps *
+gst_tensor_converter_get_template_caps (GstTensorConverter * self)
+{
+  GstCaps *caps = gst_caps_new_empty ();
+  const NNStreamerExternalConverter *ex;
+  gchar *plugin_name;
+  guint total, i;
+  subplugin_info_s info;
+  const gchar *prefix_str;
+  gsize prefix_len, extension_len, len;
+
+  /* append default caps */
+  append_video_caps_template (caps);
+  append_audio_caps_template (caps);
+  append_text_caps_template (caps);
+  append_octet_caps_template (caps);
+
+  /* get template caps from sub-plugins */
+  total = nnsconf_get_subplugin_info (NNSCONF_PATH_CONVERTERS, &info);
+  if (total > 0) {
+    prefix_str = nnsconf_get_subplugin_name_prefix (NNSCONF_PATH_CONVERTERS);
+    prefix_len = strlen (prefix_str);
+    extension_len = strlen (NNSTREAMER_SO_FILE_EXTENSION);
+
+    for (i = 0; i < total; i++) {
+      len = strlen (info.names[i]) - prefix_len - extension_len;
+      plugin_name = g_strndup (info.names[i] + prefix_len, len);
```

Comment:
Q. is this intended to do `g_size + gchar *` ?

Reply:
For the flatbuffer, the full name of lib file is `libnnstreamer_converter_flatbuf.so`.
This line gets plugin name as `flatbuf`.

Reply:
Got it. Thanks :)